<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frecuencia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duty Cicle</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>149</y>
      <width>41</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>149</y>
      <width>41</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="prueba">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>330</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setDc(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setDc(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFreq(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFreq(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prueba</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFreq(int)</slot>
  <slot>setDc(int)</slot>
  <slot>setTimer()</slot>
  <slot>closed()</slot>
 </slots>
</ui>
